docs/minikube_start.md

```diff
@@ -21,7 +21,7 @@ minikube start
       --insecure-registry=[]: Insecure Docker registries to pass to the Docker daemon
       --iso-url="https://storage.googleapis.com/minikube/minikube-0.5.iso": Location of the minikube iso
       --memory=1024: Amount of RAM allocated to the minikube VM
-      --vm-driver="virtualbox": VM driver is one of: [virtualbox vmwarefusion kvm xhyve]
```

Comment:
This doesn't look quite right, it looks like the docs generator picked up the linux drivers. Maybe switch that help text to a static list, and add a note explaining that it's platform specific?